lib/maze/internal/fixture_config.rb

```diff
@@ -0,0 +1,27 @@
+require 'json'
+
+module Maze
+  module Internal
+    # Handles the Maze Runner configuration file for test fixtures
+    class FixtureConfig
+      DEVICE_FILENAME = 'fixture_config.json'
+      WORKING_DIRECTORY = 'maze_working'
+      WORKING_FILE = File.join(WORKING_DIRECTORY, DEVICE_FILENAME)
+
+      def initialize
+        @config = {}
+      end
+
+      def add(key, value)
+        @config[key] = value
+      end
+
+      def write
+        FileUtils.makedirs(WORKING_DIRECTORY)
```

Comment:
If we're just making a file to push to the device and don't actually care about it, I think this should make a temporary file instead of cluttering up the working directory (could always log it with DEBUG if you want it visible for debugging)

Maybe `FileManager` could then take responsibility for creating the file, so you can just ask it to push a string to a device?

Reply:
In fact it's not necessary to write to a file at all - the Appium API simply accepts a string.  I've now refactored to take that into account, removing a bunch of code.